wsgi/odyssy/odyssy/urls.py

```diff
@@ -30,5 +30,9 @@
     url(r'^news/', include('news.urls')),
     url(r'^tag/', include('tag.urls')),
     url(r'^announcement/', include('announcement.urls')),
+    url(r'.html/$', include('serve_static.urls')),
```

Comment:
Those files would go in the `basic` app. And as of now, this PR should not be adding them.